sdk/keyvault/azure-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/Administrator.java

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.certificates.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public final class Administrator {
```

Comment:
Missing class level javadoc here and in a few other model classes.

Reply:
added class lvel javadoc to all public classes.